GTG/gtk/application.py

```diff
@@ -76,16 +74,11 @@ class Application(Gtk.Application):
     delete_task_dialog = None
     edit_tag_dialog = None
 
-    def __init__(self, app_id, debug):
+    def __init__(self, app_id=None):
```

Comment:
Why make app_id optional?

Reply:
I don't remember anymore... I'll make it mandatory